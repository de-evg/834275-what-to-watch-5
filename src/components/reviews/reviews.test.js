import React from "react";
import renderer from "react-test-renderer";
import Reviews from "./reviews";

const movieReview = {
  text: `Some text`,
  date: new Date(`2019-05-08T14:13:56.569Z`),
  author: `Author Name`,
  userRating: `10`
};

const movieReviews = new Array(10).fill(movieReview);

it(`Should Reviews render correctly`, () => {
  const tree = renderer
    .create(<Reviews
      movieReviews={movieReviews}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
