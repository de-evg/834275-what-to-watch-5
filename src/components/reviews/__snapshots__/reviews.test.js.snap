// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Reviews render correctly 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Some text
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Author Name
          </cite>
          <time
            className="review__date"
            dateTime="2020-10-19"
          >
            November 19, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        10
      </div>
    </div>
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Some text
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Author Name
          </cite>
          <time
            className="review__date"
            dateTime="2020-10-19"
          >
            November 19, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        10
      </div>
    </div>
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Some text
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Author Name
          </cite>
          <time
            className="review__date"
            dateTime="2020-10-19"
          >
            November 19, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        10
      </div>
    </div>
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Some text
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Author Name
          </cite>
          <time
            className="review__date"
            dateTime="2020-10-19"
          >
            November 19, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        10
      </div>
    </div>
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Some text
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Author Name
          </cite>
          <time
            className="review__date"
            dateTime="2020-10-19"
          >
            November 19, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        10
      </div>
    </div>
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Some text
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Author Name
          </cite>
          <time
            className="review__date"
            dateTime="2020-10-19"
          >
            November 19, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        10
      </div>
    </div>
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Some text
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Author Name
          </cite>
          <time
            className="review__date"
            dateTime="2020-10-19"
          >
            November 19, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        10
      </div>
    </div>
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Some text
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Author Name
          </cite>
          <time
            className="review__date"
            dateTime="2020-10-19"
          >
            November 19, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        10
      </div>
    </div>
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Some text
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Author Name
          </cite>
          <time
            className="review__date"
            dateTime="2020-10-19"
          >
            November 19, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        10
      </div>
    </div>
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Some text
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Author Name
          </cite>
          <time
            className="review__date"
            dateTime="2020-10-19"
          >
            November 19, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        10
      </div>
    </div>
  </div>
</div>
`;
